isis/src/control/objs/ControlPoint/ControlPoint.cpp

```diff
@@ -306,7 +307,7 @@ namespace Isis {
 
     // notify parent network of the change
     if (parentNetwork) {
-      parentNetwork->measureDeleted(cm);
+      parentNetwork->emitMeasureRemoved(cm);
```

Comment:
This need to go back to measureDeleted. The emit call doesn't update the graph and measureDeleted will emit the same signal still.